src/Movies/Movies.js

```diff
@@ -0,0 +1,20 @@
+import react from "react"
+import "./Movies.css"
+
+const Movies = ({movies, displaySingleMovie}) => {
+    const moviePoster = movies.map(movie => {
+        return (
+           <img key={movie.id} src={movie.poster_path} onClick={() => displaySingleMovie(movie.id)} height="100px"/>
+        )
+    })
```

Comment:
Nice job keeping this logic out of your component's return statement.  We want each component's return statement to be as clean, simple and logic-free as possible.  Keep this up!